specification/cosmos-db/resource-manager/readme.md

```diff
@@ -42,6 +42,7 @@ input-file:
   - Microsoft.DocumentDB/stable/2021-10-15/privateLinkResources.json
   - Microsoft.DocumentDB/stable/2021-10-15/rbac.json
   - Microsoft.DocumentDB/stable/2021-10-15/restorable.json
+  - Microsoft.DocumentDB/stable/2021-10-15/mongorbac.json
```

Comment:
From the files, I see that you are only adding the changes to the preview version. So this should not be necessary.

Reply:
Updated.